basetest.pm

```diff
@@ -387,8 +387,10 @@ sub runtest {
         $self->run_post_fail("test $name failed");
     }
 
+    my $execution_time = time - $starttime;
+    $self->{details}->{execution_time} = $execution_time . ' s';
```

Comment:
I feel like I'd expect the value to be the seconds as a number. Turning it into a string for display purposes belongs on the display side of things.

Reply:
fixed

Reply:
Thanks!